test/TritonGPU/amd/amd-instruction-sched.mlir

```diff
@@ -1,14 +1,12 @@
-// RUN: triton-opt %s -triton-amdgpu-insert-instruction-sched-hints='variant=llvm_iglp_0' -triton-amdgpu-lower-insert-instruction-sched-hints -verify-diagnostics | FileCheck %s -check-prefix=INSERT_IGLP0
-// RUN: triton-opt %s -triton-amdgpu-insert-instruction-sched-hints='variant=llvm_iglp_1' -triton-amdgpu-lower-insert-instruction-sched-hints -verify-diagnostics | FileCheck %s -check-prefix=INSERT_IGLP1
+// RUN: triton-opt %s -triton-amdgpu-insert-instruction-sched-hints='variant=attention' -triton-amdgpu-lower-insert-instruction-sched-hints -verify-diagnostics | FileCheck %s -check-prefix=INSERT_ATTENTION
```

Comment:
This test file is quite cluttered and we might want to remove it later given the local prefetch experiement is not particularly fruitful. Can you use a new test file to have a more targeted test case?